ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -57,11 +57,22 @@ message Commands {
   // Required
   repeated Command commands = 8;
 
-  // The length of the time window during which all commands with the same ``act_as`` parties and
-  // the same command ID will be deduplicated.
-  // Duplicate commands submitted before the end of this window return an ALREADY_EXISTS error.
-  // Optional
-  google.protobuf.Duration deduplication_time = 9;
+  oneof deduplication {
```

Comment:
I thought we were going with `deduplication_period`?

Might be wrong, but I definitely find it more understandable.

Reply:
Agree, deduplication_period it is